docs/runbooks/CDIDefaultStorageClassDegraded.md

```diff
@@ -0,0 +1,51 @@
+# CDIDefaultStorageClassDegraded
+<!-- Edited by agilboa, 6 Dec 2023 -->
+
+## Meaning
+
+This alert fires when the default (Kubernetes or virtualization) storage class supports smart clone (either CSI or snapshot based) and ReadWriteMany.
```

Comment:
Is this first sentence correct? It sounds like something is missing. Should it be "This alert fires when the default storage class does NOT support smart clone and RWX"?

Reply:
Right... Fixing in a follow-up.